homeassistant/components/mqtt/climate.py

```diff
@@ -309,7 +306,7 @@ def valid_preset_mode_configuration(config):
         vol.Optional(CONF_SWING_MODE_COMMAND_TEMPLATE): cv.template,
         vol.Optional(CONF_SWING_MODE_COMMAND_TOPIC): mqtt.valid_publish_topic,
         vol.Optional(
-            CONF_SWING_MODE_LIST, default=[STATE_ON, HVAC_MODE_OFF]
+            CONF_SWING_MODE_LIST, default=[SWING_ON, SWING_OFF]
```

Comment:
I think this was a bug...

Reply:
Yes, looks like it.  And the underlying strings are "on"/"off" anyway in both cases, so this change should make no difference.